pkg/apis/monitoring/v1/prometheus_types.go

```diff
@@ -1655,6 +1655,11 @@ type AlertmanagerEndpoints struct {
 	//
 	// +optional
 	EnableHttp2 *bool `json:"enableHttp2,omitempty"`
+
+	// List of alertmanager relabel configs
+	//
+	// +optional
+	RelabelConfigs []*RelabelConfig `json:"relabelConfigs,omitempty"`
```

Comment:
```suggestion
	RelabelConfigs []RelabelConfig `json:"relabelings,omitempty"`
```

Reply:
I think we can change the type after we merge #6479?